### DISCUSSION TITLE
Meaning of plasma protein scaling factor in the context of macromolecules

### DISCUSSION THREAD

**Discussion by RobinM92** [Open-Systems-Pharmacology/Forum#984] [Questions & Problems]
Dear community,

I am currently working on a pre-clinical PBPK model for a therapeutic protein (including IgG) after IV administration. Using parameters based on in vitro data I obtained a descent description of the Cmax, elimination and the half life, but not of the initial distribution, thus leading to overprediction of the concentration-time profile. Doing a sensitivity analysis showed (not surprisingly) Kd(FcRn) and the solute radius to be important, as well as the plasma protein scaling factor. Running parameter identification with those parameters leads to a perfect fit but also to a value of 20 for the PPSF, which seems to be very high? How would we interpret this value? Or would this point to something else being wrong/missing in the model? 

Thank you very much for your insights!

**Comment by prvmalik** [comment]
Plasma protein scaling factor scales fraction unbound. Fraction unbound should be 1, and the plasma protein scaling factor should not be different from 1 in a model for a therapeutic protein.

**Reply by StephanSchaller** [reply]
I agree, except if you have a protein designed to bind to plasma protein?

**Comment by StephanSchaller** [comment]
As in the previous answer, PPSF should not have a large influence if fu = 1. Can you repeat the fit without fitting PPSF and check if you can still get an optimal fit?
What is your protein binding to? Any specific targets (abundantly / highly expressed? Any TMDD?), ...?

**Reply by RobinM92** [reply]
Dear Stephan,

Not adding PPSF to the identification leads to a suboptimal fit, as the other sensitive parameters (Kd FcRn and solute radius) also impact the terminal phase and thus either the initial decline is predicted well with a large mismatch of the terminal phase or vice versa.

**Reply by StephanSchaller** [reply]
I would assume solute radius has a stronger impact on the distribution phase with Kd FcRn having a stronger impact on the terminal phase.
PPSF basically scales your fu (an thereby mostly the extravasation rate). Reducing solute radius should have a similar effect.

**Comment by RobinM92** [comment]
Thanks Stephan and Paul for your insights,

I agree that PPSF should stay 1 in this kind of setting, which is why I was so surprised by the results and I assume there is something else that I am missing. The therapeutic protein has a target-binding domain and an IgG-Fc part (thus binding to FcRn), but these data are from animals where the target is not present. Perhaps another binding partner is present that has not been considered before, but I would expect this to give lower PPSF rather than higher?
Could it be that the administration of the compound leads to some shifts in blood volumes that skew the first measurements? Administered volume is between 100 and 200 µL in mice, so up to 10% of the blood volume.

**Reply by StephanSchaller** [reply]
Yes, it might be, that the administered volume increases extravasation flux (via convection)?

**Reply by RobinM92** [reply]
Interesting, I assume we would go to MoBi to test that? Or is there a parameter that could be tweaked to simulate this?

**Reply by StephanSchaller** [reply]
The effect would likely be transient (minutes to hours). So technically, yes, you would have to do this in MoBi to test it. But this will not be trivial.

**Reply by RobinM92** [reply]
Yes, e.g. changing venous plasma volume to reflect the change of course shifts the profile in the correct direction, but would then be fixed for the entire duration. Another option would be to ignore the early time points and see whether I can get the model to predict the concentrations after e.g. 8h well. But that is then assuming that the transient effect would have no impact on the subsequent concentrations.

@Christoph27 do you have any insights on how to impact the extravasation (due to a large dosed volume in the vein)?

**Comment by Christoph27** [comment]
Dear Robin, 
you write that you have a  therapeutic protein "including IgG". Could you indicate how large your protein is?

**Reply by RobinM92** [reply]
Hi Christoph,
Sure, it has a size of 235 kDa.

**Reply by prvmalik** [reply]
and just making sure, that you have "is small molecule" unchecked in the compound building block. sometimes I forget this too :)

**Reply by RobinM92** [reply]
That is also the case, but indeed good to check!

**Reply by RobinM92** [reply]
@prvmalik Do you perhaps think that a similar approach as you applied in your PhD to increase the 'leakiness' and thus extravasation in preterm infants could be applied to model an increased extravasation flux due to the high fluid given with an IV dose?

**Reply by prvmalik** [reply]
Yes you could, but it involves going into Mobi. You can add a scaling factor (e.g., the conventional 'F2') on the convective flow term in the "Two Pore Transport" passive diffusion process. It is unlikely to change the initial distribution volume and has meaningful effects on the rate and extent of distribution at around 100 hours on the PK profile. Trough and elimination typically remain unchanged post 500 hours.

Another thing I did in the infliximab paper was to increase vascular pore size to describe increased extravasation into inflamed tissues, such as the gut in Crohn's disease or ulcerative colitis.

**Reply by RobinM92** [reply]
I'll check that out! But this would still not be a transient effect right?
Perhaps something similar as an ontogeny function while enabling aging would be feasible? Changing e.g. the convective flow term from something high to its normal level in a the matter of hours?

**Reply by StephanSchaller** [reply]
As mentioned you could trigger this with an event with your administration

**Reply by RobinM92** [reply]
@StephanSchaller I might have missed that answer?

**Reply by RobinM92** [reply]
Or was this meant for the fasted/fed discussion?

**Reply by prvmalik** [reply]
@RobinM92 what do you mean by transient? changing convective flow can change the distribution phase of the profile while leaving the terminal phase largely unchanged. Just depends if that is the correct amount of transience you are looking for.

**Reply by RobinM92** [reply]
Hi Paul (and others),
I tried out adapting the convective flow in Mobi by addition of a scaling factor F2, and then applied a PI on it, leading to a F2 of 18, which seems again very high (comparable to the PBSF which started this discussion).

This is the PBPK prediction without any parameter identification (left out the units, but it is on a log scale and 4 week time span)

![image](https://user-images.githubusercontent.com/30694339/138773666-bb7bfdb7-2ebb-439e-af0d-7d4faa049f33.png)

And this is the fit with a F2 scaling factor of 18, added to the convective flow eq in MoBi:

![image](https://user-images.githubusercontent.com/30694339/138773581-930154ec-c261-4fef-90b6-ddbd19e12b45.png)

Would it be reasonable to attribute this large discrepancy to injection volume related distrubances or am I missing something else?
Thanks a lot!

All the best,
Robin

**Reply by RobinM92** [reply]
> 
> 
> Hi Paul (and others), I tried out adapting the convective flow in Mobi by addition of a scaling factor F2, and then applied a PI on it, leading to a F2 of 18, which seems again very high (comparable to the PBSF which started this discussion).
> 
> This is the PBPK prediction without any parameter identification (left out the units, but it is on a log scale and 4 week time span)
> 
> ![image](https://user-images.githubusercontent.com/30694339/138773666-bb7bfdb7-2ebb-439e-af0d-7d4faa049f33.png)
> 
> And this is the fit with a F2 scaling factor of 18, added to the convective flow eq in MoBi:
> 
> ![image](https://user-images.githubusercontent.com/30694339/138773581-930154ec-c261-4fef-90b6-ddbd19e12b45.png)
> 
> Would it be reasonable to attribute this large discrepancy to injection volume related distrubances or am I missing something else? Thanks a lot!
> 
> All the best, Robin

Dear all,

I was wondering if there are other approaches I could try here or whether I just need to accept the very large F2/PPFS? 

Thank you very much!
Kind regards,
Robin

**Reply by prvmalik** [reply]
I agree an 18 would not be physiologically defensible.

What solute radius did you get? Technically the models are not calibrated to A) predict accurate solute radius past 200 kDa or B) represent distribution through the existing pore sizes for molecules greater than 6 nm radius.

**Comment by RobinM92** [comment]
Hi Paul,

That would be exactly what I am looking for, I will try it out asap.
To confirm, that would be like adding a factor F2 on the convective flow 
in the two pore diffusion model right?

Dr. ir. Robin Michelet
Senior scientist

Freie Universitaet Berlin
Institute of Pharmacy
Dept. of Clinical Pharmacy & Biochemistry
Kelchstr. 31
12169 Berlin
Germany
Phone:  + 49 30 838 50659
Fax:      + 49 30 838 4 50656
***@***.***
www.clinical-pharmacy.eu
https://fair-flagellin.eu/

**Comment by RobinM92** [comment]
Dear Paul,

Thanks again for your answer! The compound indeed is a borderline case: 
MW of 230-235 kDa and predicted solute radius of 6.17 nm. After PI the 
radius is estimated to be 7.5 nm.

Does this mean I cannot use the two-pore model or would some 
modifications/calibrations be possible? I have mouse and hamster data 
available and NHP data on its way.

Kind regards,

Robin

--
Dr. ir. Robin Michelet
Senior scientist

Freie Universitaet Berlin
Institute of Pharmacy
Dept. of Clinical Pharmacy & Biochemistry
Kelchstr. 31
12169 Berlin
Germany
Phone:  + 49 30 838 50659
Fax:      + 49 30 838 4 50656
Email: ***@***.***
www.clinical-pharmacy.eu
https://fair-flagellin.eu/

Am 04/11/2021 um 17:04 schrieb Paul Malik:

**Reply by Christoph27** [reply]
Blood volume and extracellular volume are tightly regulated; thus, I won't think that any effect from the additional volume from the application has a long effect. Can you rule out binding of the drug to a e.g. a target?  What I do not understand: You overpredict the plasma concentrations but after the PI your solute radius is higher? Should be the other way round ….

**Comment by RobinM92** [comment]
Hi everyone,

I would like to update you regarding this topic if you would be interested.
After all discussions here and having an implausibly high PPSF in PKSim or F2 estimate in MoBi, I went back to the compound itself and took a.o. @Christoph27 's comment to heart. As stated originally the compound includes IgG in its structure, but this is not the only part and until now only antibody-related processes were considered in the model. The inclusion of a non-specific binding process, assumed to be related to other parts of the molecule, completely solved the misfit, allowing for a rapid decrease in plasma concentrations after administration after which the PK starts following expected antibody-like kinetics. So now it's back to the lab to figure if this binding partner would indeed be related to the target or is something else entirely. There are some plausible pistes to follow which were not considered before, again showcasing the beautiful hypothesis-generating power of PBPK modeling :)

Thanks all for you input! 
Robin

**Reply by StephanSchaller** [reply]
Nice, keep us updated! ;-)

**Reply by Schwartzenegger** [reply]
Hi Robin,
I read this discussion with interest and realize your main issue is with the peripheral clearance. 
Still, might it be so that your last timepoint in the observed PK has suffered from many samples below the limit of quantification? (I am asking because the last time point does not align with the rest of time points in the terminal phase)
Then the remaining samples may cause an upward bias in the observed mean and consequently the best fit may underestimate the central clearance. Would you advise to correct for that or is it likely of minor importance?
thanks, Arnie
